src/Datamap.js

```diff
@@ -0,0 +1,279 @@
+import d3 from 'd3';
+import topojson from 'topojson';
+import Datamaps from 'datamaps/dist/datamaps.usa.min.js';
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+class Datamap extends Component {
+  constructor(){
+    super();
+  }
+  // this.datamap = null;
+    // resize() {
+    // window.addEventListener('resize', this.resize);
+    //     if (this.map) {
+    //         this.map.resize();
+    //     }
+    // }
+    // componentDidMount() {
+    //     this.drawMap();
+    // }
+    // 
+    // componentWillReceiveProps() {
+    //     this.clear();
+    // }
+    // 
+    // componentDidUpdate() {
+    //     this.drawMap();
+    // }
+    // 
+    // componentWillUnmount() {
+    //     this.clear();
+    //     window.removeEventListener('resize', this.resize);
+    // }
+    // 
+    // clear() {
+    //     const container = this.refs.container;
+    // 
+    //     for (const child of Array.from(container.childNodes)) {
+    //         container.removeChild(child);
+    //     }
+    // }
+
+  drawMap(){
+    var map = new Datamap({
```

Comment:
I think it's that you're using `Datamap` here, but when you imported the library on line 3 it was named `Datamaps`. So this line would be referencing the React component instead of the library.

I also cloned and ran your branch super quick and wasn't able to find an element with the id `map_election` but wasn't sure if I was missing something. Hope this is helpful!